test/e2e/kubectl/kubectl.go

```diff
@@ -453,6 +449,14 @@ var _ = SIGDescribe("Kubectl client", func() {
 			testableResources := etcd.GetEtcdStorageDataForNamespace(f.Namespace.Name)
 
 			for _, group := range apiGroups {
+				// This significantly limits the scope of this test, but should
+				// ensure it's only covering Kubernetes API resources and not
+				// CRDs or other unexpected resources. For coverage of all
+				// resources in a cluster, remove this line.
+				if !strings.Contains(group.GroupVersion, ".k8s.io") {
+					continue
```

Comment:
Won't it exclude e.g. core/v1 ?

Also - someone from SIG CLI and/or Apimachinery should remove this one.

Reply:
Yep, this assumes no new API resources will be added to that group. Definitely not ideal, but was trying to avoid adding a long list of approved API groups here. The real goal of this test is to ensure that new API resources are added with kubectl get output that works as intended.